articles/libraries/lock/v11/migration-guide.md

```diff
@@ -41,7 +41,7 @@ If you have any questions or concerns, you can discuss them in the [Auth0 Commun
 
 ## Disabling legacy Lock API
 
-After you update to Lock v11 and/or Auth0.js v9, it is advised that you turn off the **Legacy Lock API** toggle in the Dashboard. This will make your Auth0 tenant behave as if the legacy API is no longer available. Starting on July 16, 2018, this option will be forcibly disabled, so it is recommended you opt-in before that time to verify that your configuration will work correctly. 
+After completing the migration to the latest versions, make sure that you turn off the **Legacy Lock API** toggle in the Dashboard. This will make your Auth0 tenant behave as if the legacy API is no longer available. Starting on **July 16, 2018**, this option will be forcibly disabled, so it is recommended you opt-in before that time to verify that your configuration will work correctly. Additionally, the `/usernamepassword/login` and `/ssodata` endpoints will be removed from service on **July 16, 2018**. These are used by Lock.js v8, v9, v10, and auth0.js v6, v7 and v8.
```

Comment:
Again, not sure about adding this part: "Additionally, the `/usernamepassword/login` and `/ssodata` endpoints will be removed from service on **July 16, 2018**. These are used by Lock.js v8, v9, v10, and auth0.js v6, v7 and v8.". I'd like hilary.elgin@auth0.com to review (don't know her github user). Maybe @Annyv2 or @aaguiarz  can help ??

Otherwise let's exclude from this PR so we can release the production checks. Thx!

Reply:
Hilary suggested @jeffreylees to review.

Reply:
Let's not add this last line about the endpoints - it's extraneous information since this warning is within the Lock 11 migration guide. That guide only has the one point - to migrate Lock versions.

Reply:
Done.